css-forms-1/Overview.bs

```diff
@@ -696,6 +696,51 @@ input[type=radio]:checked::checkmark {
     width: 100%;
 }
 ```
+
+## Text fields
+
+```css
+input:not([type]),
+input:is([type=text], [type=search], [type=email], [type=password], [type=tel], [type=url], [type=number]),
+textarea {
+  display: inline-grid;
+  padding-inline: 2px;
+}
+
+input:not([type]),
+input:is([type=text], [type=search], [type=email], [type=password], [type=tel], [type=url], [type=number]) {
+  overflow-x: auto;
+  scrollbar-width: none;
+  padding-block: 1px;
+}
+
+textarea {
+  overflow-y: auto;
+  padding-block: 2px;
+  resize: both;
+}
+
+::placeholder {
+  color: color-mix(in srgb, currentColor 50%, transparent);
+  display: none;
+  line-break: anywhere;
+  grid-area: 1/1;
+}
+
+:placeholder-shown::placeholder {
+  display: block;
+}
+
+::field-text {
+  grid-area: 1/1;
+  min-block-size: 1lh;
+  /* Should be cursor width but that's not exposed */
+  min-inline-size: 1px;
```

Comment:
Should this actually be 24px? So the textarea/input is never smaller than 24px x 24px?

Reply:
I wonder if this really belongs on field-text, ditto for min-block-size/font-size. Ideally they'd just inherit what they need from the parent.